+exp/ExpTest.m

```diff
@@ -51,8 +59,9 @@
     ExpDef % function handle to currently chosen Signals Exp Def
     ExpDefPath % fullfile path of ExpDef
     OptionsButton % handle to 'Options' push-button
-    LivePlot = 1 % option for live-plotting signals during experiment
+    LivePlot = 0 % option for live-plotting signals during experiment
     SaveBlock = 0 % option for saving 'block' file at experiment end
+    ScreenView = 0 % option for viewing PTB window as a single screen
```

Comment:
I'd give this a more informative name.  If this is a choice between simulating 3 vs 1 screen, call it something like 'SingleScreen'